Base/res/js/Spreadsheet/runtime.js

```diff
@@ -147,6 +147,18 @@ class Ranges {
         }
     }
 
+    at(wantedIx) {
+        let ix = 0;
+        let found = null;
+        this.forEach(cell => {
```

Comment:
does libjs support something like `Array.prototype.findIndex` or `Array.prototype.find`?

Reply:
where would you get the array to do it on?